configs/dino/metafile.yml

```diff
@@ -48,3 +48,15 @@ Models:
     Results:
       - Task: Object Detection
         Dataset: COCO
+
+  - Name: dino-5scale_swin-l_8xb2-12e_coco.py
```

Comment:
Great! Would you like to update the README.md? BTW, would you like us to host your model weights on our servers? You can indicate your ownership in the readme and model weight names.

Reply:
I would like to host on your servers.

Reply:
Would you like to retrain it with convnext v2？
https://github.com/LYMDLUT/mmdetection/blob/3.x/projects/ConvNeXt-V2/configs/dino-4scale_r50_8xb2-12e-fcmae-atto_coco.py

Reply:
maybe you can take it as a reference

Reply:
@LYMDLUT I will try it later.